### PR TITLE
Edit UI colours

### LINKED ISSUE #92: Background colour and PersonCard UI

### PR DESCRIPTION
**Pull request by waihin26**
Fixes #92

### CODE REVIEW

File: src/main/resources/view/EdututuTheme.css [hunk 1]
```diff
@@ -77,84 +94,93 @@
...
     -fx-border-width: 1;
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color: derive(-ab3-background-2, 20%);
 }
 
 .list-view {
     -fx-background-insets: 0;
     -fx-padding: 0;
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color: #D2C0BD;
```
Comment (sumomomomomo) [inline src/main/resources/view/EdututuTheme.css:110]: Remove magic literals

File: src/main/resources/view/EdututuTheme.css [hunk 2]
```diff
@@ -77,84 +94,93 @@
...
+    -fx-border-color: #D2C0BD; /* Light color for the border */
+    -fx-border-width: 0 0 4px 0; /* Adds a thin line only at the bottom */
+    -fx-label-padding: 10 10 10 10; /* Adds some padding for better spacing */
 }
 
+
+
 .list-cell:filled:even {
-    -fx-background-color: #3c3e3f;
+    -fx-background-color: #FAFAF5;
```
Comment (sumomomomomo) [inline src/main/resources/view/EdututuTheme.css:128]: Ditto

**Review by sumomomomomo — COMMENTED** [review]
Good apart from some magic literals

File: src/main/resources/view/EdututuTheme.css [hunk 3]
```diff
@@ -175,31 +201,31 @@
...
 .menu-bar {
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color: derive(-ab3-background-2, 20%);
 }
 
 .menu-bar .label {
     -fx-font-size: 14pt;
     -fx-font-family: "Segoe UI Light";
-    -fx-text-fill: white;
+    -fx-text-fill: #584747;
```
[the remark this answers is not in the document]
Comment (waihin26) [reply src/main/resources/view/EdututuTheme.css:228]: Fixed

File: src/main/resources/view/EdututuTheme.css [hunk 4]
```diff
@@ -77,84 +94,93 @@
...
+    -fx-background-color: derive(-ab3-background-2, 30%);
 }
 
 .result-display {
-    -fx-background-color: transparent;
+    -fx-background-color: #F0E7E6;
     -fx-font-family: "Segoe UI Light";
     -fx-font-size: 13pt;
-    -fx-text-fill: white;
+    -fx-text-fill: #333333; /* Darker text for better visibility */
```
[the remark this answers is not in the document]
Comment (waihin26) [reply src/main/resources/view/EdututuTheme.css:178]: Fixed

File: src/main/resources/view/EdututuTheme.css [hunk 5]
```diff
@@ -77,84 +94,93 @@
...
 }
 
 .status-bar {
-    -fx-background-color: derive(#1d1d1d, 30%);
+    -fx-background-color: derive(-ab3-background-2, 30%);
 }
 
 .result-display {
-    -fx-background-color: transparent;
+    -fx-background-color: #F0E7E6;
```
[the remark this answers is not in the document]
Comment (waihin26) [reply src/main/resources/view/EdututuTheme.css:175]: fixed

File: src/main/resources/view/EdututuTheme.css (hunk 1 above)
Comment (waihin26) [reply src/main/resources/view/EdututuTheme.css:110]: fixed

**Review by hooyushing — COMMENTED** [review]
LGTM

**Pull request merged**